src/Resources/skeleton/forgottenPassword/PasswordResetToken.tpl.php

```diff
@@ -0,0 +1,87 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace ?>;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class <?= $class_name ?>
+
+{
+    const TTL_HOURS = 24;
+    const SELECTOR_LENGTH = 20; // in chars
+
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $selector;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $token;
+
+    /**
+     * @ORM\Column(type="datetime")
```

Comment:
If we use `datetime_immutable`, we need to bump `doctrine/orm` required version to `~2.6`. I thought about it, but I thought it may be too restrictive :S

Reply:
Ah, yes, then we can skip it for this PR 👍 But probably in a separate PR with some extra check we can figure out Doctrine version and use `datetime_immutable` if it's `2.6`

Reply:
Yes, it sounds like a nice enhancement!

Reply:
versions of Doctrine older than 2.6 are not maintained anymore, so that's fine IMO.

Reply:
Ok, definitely I prefer `datetime_immutable` :)